components/script/dom/webglrenderingcontext.rs

```diff
@@ -1084,6 +1084,48 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             .unwrap()
     }
 
+    // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.3
+    fn StencilFunc(&self, func: u32, ref_: i32, mask: u32) {
```

Comment:
I think these should validate the `func` parameter as described [here](https://www.khronos.org/opengles/sdk/docs/man/xhtml/glStencilFunc.xml)